src/js/rtc_session.js

```diff
@@ -477,13 +477,82 @@ export default class RtcSession {
     get sessionReport() {
         return this._sessionReport;
     }
-
     get callId() {
         return this._callId;
     }
     get mediaStream() {
         return this._userAudioStream;
     }
+    get localVideoStream() {
```

Comment:
resolved. In the most-recent commit, it supports client provided stream. Docs added as well.